matchzoo/utils/bert_utils.py

```diff
@@ -0,0 +1,104 @@
+import unicodedata
+
+
+def is_whitespace(char):
+    """Checks whether `chars` is a whitespace character."""
+    # \t, \n, and \r are technically contorl characters but we treat them
+    # as whitespace since they are generally considered as such.
+    if char == " " or char == "\t" or char == "\n" or char == "\r":

```

Comment:
```python
or ... or ... or unicodedata.category(char) == 'Zs':
    return True
return False
```

And mroeve 11 - 12, more readable.

Reply:
Because of the codestyle constraints about the if condition format, I change it as follow:
``return (char == " ") or \
           (char == "\t") or \
           (char == "\n") or \
           (char == "\r") or \
           (unicodedata.category(char) == "Zs")``
With the constraint of length, each token `\` follows a break line and the parentheses `()` is added to avoid codestyle error.